CONTRIBUTING.md

```diff
@@ -31,7 +33,18 @@ Participation in this project comes under the [Contributor Covenant Code of Cond
   guide for details.
 - All submissions, including submissions by project members, will require review before being merged.
 - Once review has occurred, please rebase your PR down to a single commit. This will ensure a nice clean Git history.
-- Finally - *Thanks* for considering submitting code to Agones!
+- Please follow the code formatting instructions below.
+
+### Additional Instructions for Unreal Plugin Pull Requests
+
+As there is no CI for the Unreal plugin, the following checklist should be run
+manually before the PR is approved, using the latest released version of UE4.
+
+1. Create default C++ template project in UE4.
+1. Create a Plugins folder under the project directory (should be a sibling of the .uproject file).
+1. Copy the [sdks/unreal/Agones](sdks/unreal/Agones) directory into the Plugins folder.
+1. Build the UE4 project.
+1. If the build succeeded, paste the build logs into the PR.
```

Comment:
Nice, think this last step is really good, showing that it has been run and verified. :D